src/AutomationTests/SnapshotCommandUnitTests.cs

```diff
@@ -259,13 +205,15 @@ public void Execute_UpperBoundIsReached_ThrowsAutomationException_ErrorAutomatio
                         new A11yElement()),
                 };
 
-                InitializeShims(populateLocationHelper: false, enableRetention: false, shimTargetElementLocator: true,
+                InitializeShims(shimTargetElementLocator: true,
                     selectAction: selectAction, elementBoundExceeded: true, shimUiFramework: true,
                     setTestModeSucceeds: true);
 
+                var config = Config.Builder.ForProcessId(-1).Build();
+
                 try
                 {
-                    SnapshotCommandResult result = SnapshotCommand.Execute(new Dictionary<string, string>());
+                    SnapshotCommandResult result = SnapshotCommand.Execute(config, OutputFileHelperStub);
                 }
                 catch (System.Exception ex)
```

Comment:
"System." is not needed here

```suggestion
                catch (Exception ex)
```